### PR TITLE
Added "Senior Product Developer - Data Farming" JD

### PR DESCRIPTION
**Pull request by LoznianuAnamaria**
Changes proposed in this PR:

- Added "Senior Product Developer - Data Farming" JD
- Added link readme to the new DF JD

### CODE REVIEW

**Review by DraganaRajacic — APPROVED** [review]

File: df.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* Over time they will ideally automate/delegate parts or sections of the program for optimization. Some things can be automated using script and tech. They will design initiatives to decentralize other decision-making efforts to the community via panel or voting.
+
+
+
+## Skills and Experience 
+* We don’t expect applicants to be an expert in everything listed below which is why we’ve separated the requirements into two tiers. The ideal candidate loves to communicate, to code, to build meaningful technical communities and relationships with key stakeholders.
+
+
+## Minimum Qualifications
+* +3 years of experience working in technical/development roles in the crypto projects
```
Comment (DraganaRajacic) [inline df.md]: 3+ years of experience

File: df.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* +2 years Hands-on experience in development in Web3 and Web2 tech stack
```
Comment (DraganaRajacic) [inline df.md]: 2+ years Hands-on experience

**Review by DraganaRajacic — APPROVED** [review]

**Comment by kremalicious** [comment]
Developer vs. Engineer. If we mean engineer here then please use engineer. Never heard of a "Product Developer", it's "Product Manager" or "Product Owner" or "Product Designer" or "Community Manager" from reading job description but actually not really clear what's wanted here. Is it a management role? Or a coding role? Also, "executing scripts" is not something a senior engineer would be interested in, but that is the only technical task reference in the job description.

**Comment by DraganaRajacic** [comment]
The role is to evolve and optimize the program based on users engagement and community feedback. That means to measure KPI's and fine tune of rewarding function according to the users activities. That is why was Product Developer, not Engineer

**Comment by kremalicious** [comment]
So it's a Senior Program Manager or Technical Program Manager you are looking for? "Product" implies they are supposed to build a product, but data farming is no product but a program to increase usage of our actual product. Are they supposed to actively work on Ocean Market? If not, they have nothing to do with product.

As for "developer" you already see that this term is too vague and ambiguous as it can be understood as coding or not depending on who you ask. And the people you want to reach with this here might look for other things than "developer"

**Comment by trentmc** [comment]
Even if a product's main goal is to help drive adoption of another product, it can still be a product on its own. 

Data Farming has the trappings of a product: software artifacts & related processes, clear targets, a team to drive it, a life cycle (vs a project with a date that ends), iterations to keep improving it, etc. By framing as a product then we get all the tools and mental models of products. So we are going to continue calling it a product.

However none of our other job titles around software development roles have "product" in them. They don't need to. It adds confusion, as Matthias has pointed out. Therefore I recommend that we do _not_ have "product" in this job description either. 

Finally, "Blockchain" helps attract actual blockchain developers, and improves discoverability due to keyword searches for blockchain. 

We've used "Senior Blockchain Developer" a lot in the past and it's served us very well.

Therefore I recommend "Senior Blockchain Developer - Data Farming" for this one.

File: df.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+* Over time they will ideally automate/delegate parts or sections of the program for optimization. Some things can be automated using script and tech. They will design initiatives to decentralize other decision-making efforts to the community via panel or voting.
+
+
+## Skills and Experience 
+* We don’t expect applicants to be an expert in everything listed below which is why we’ve separated the requirements into two tiers. The ideal candidate loves to communicate, to code, to build meaningful technical communities and relationships with key stakeholders.
+
+
+## Minimum Qualifications
+* 3+ years of experience working in technical/development roles in the crypto projects
+* 2+ years Hands-on experience in development in Web3 and Web2 tech stack
```
Comment (trentmc) [inline df.md]: My comments are for the two bullets above: 3+ years exp crypto, 2+ years dev Web 2 & Web3

Problems:
- 3+ years working in Web3 is a big ask; it greatly narrows our candidates. Same for "2+ years working in Web3 development". It's unnecessarily constraining.
- Web2 experience is a radically different thing than Web3, it should be its own line.
- Web2 and Web3 it should be more specific on which stack. 
- Having two separate lines on Web3 is a bit confusing. If you ask for a min req on Solidity experience you don't need a separate line on more general "technical" Web3 experience

Suggested new lines:

- 6+ months Solidity experience
- 3+ years full-stack Web2 experience including NodeJS

File: df.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+* Proven experience in building meaningful technical communities around projects
+* Ability to build and manage relationships with key stakeholders
+* Ability to take calculated decisions based on data 
+* Process-driven, proactive - able to report on your activity and communicate well with the team.
+* Comfortable working autonomously and within a team - our team is small, so balancing independence and collaboration is important.
+* Creative, with extensive writing/public speaking experience
+* Highly motivated, with the ability to deal with the day to day tasks as well as the more strategic aspects of the role
+* Stellar research skills
```
Comment (trentmc) [inline df.md]: There are too many bullets for "minimum qualifications", and too few in "Nice to haves". You'll scare off candidates.

Suggestion: move some "minimum qualifications" into "nice to haves".

File: df.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+## Responsibilities
+* Candidate will have to communicate with different teams to sync on roadmap delivery
+* Candidate will need to engage with the community to get faster feedback to understand usage patterns, pain points, and user experience
+* They will need to understand to differentiate good and bad feedback and find real pain points and needed improvements
+* They will improve and evolve product/program based on learnings from usage data and user feedback 
+* They will take ownership of related tech like scripts and manage and update scripts and reward functions to implement design changes and effectively communicate to the community about changes
+* They will have to execute scripts to automate some web3 tasks like token distribution
+* They will have to identify fraudulent actors taking unfair advantage of the program and make improvements to curb them
+* Over time they will ideally automate/delegate parts or sections of the program for optimization. Some things can be automated using script and tech. They will design initiatives to decentralize other decision-making efforts to the community via panel or voting.
```
Comment (trentmc) [inline df.md]: The list of responsibilities is too long. Yet still weirdly vague.

Here's a new set, inspired by https://github.com/oceanprotocol/jobs/blob/main/sr-dev.md:
- You will design & build software for Ocean Protocol Data Farming, including scripts for reading on-chain data, performing analysis, token distribution, and whatever else is needed for the Data Farming process. The scope of the software will grow over time, based on feedback from the community, e.g. to do on-chain voting and interact with more protocols.
- You will co-plan and co-design the product with the Data Farming team in an agile, open-source development process

File: df.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+# 🦈 Senior Product Developer - Data Farming 
+[Ocean Protocol](https://www.oceanprotocol.com) builds tools for people to reclaim control of their data and unleash an open data economy. Ocean tools make it easy to publish & consume data services as data access tokens, and bridge data to DeFi. 
+
+Ocean has a vibrant, expanding ecosystem of data suppliers, data consumers, stakers, and community developers. To serve it better, we’re hiring! If you are someone that loves to build things and cares deeply about the future of data, we want to hear from you.
```
Comment (trentmc) [inline df.md]: We can make the above more specific.

How about:

Data Farming is a specific initiative in the spirit of liquidity mining, with generous OCEAN rewards to ecosystem participants who drive the supply of data. We're growing the Data Farming team. If you love to work with crypto, to build and to engage deeply with an enthusiastic community, we want to hear from you.

File: df.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+## Minimum Qualifications
+* 3+ years of experience working in technical/development roles in the crypto projects
+* 2+ years Hands-on experience in development in Web3 and Web2 tech stack
+* Proven experience in building meaningful technical communities around projects
+* Ability to build and manage relationships with key stakeholders
+* Ability to take calculated decisions based on data 
+* Process-driven, proactive - able to report on your activity and communicate well with the team.
+* Comfortable working autonomously and within a team - our team is small, so balancing independence and collaboration is important.
+* Creative, with extensive writing/public speaking experience
```
Comment (trentmc) [inline df.md]: Everyone will say they're "creative". We don't need this hear.

And we don't need extensive public speaking experience for this role.

I recommend to (a) remove this line, and (b) in "Nice to haves" add "You’re an excellent writer."

File: df.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+* Highly motivated, with the ability to deal with the day to day tasks as well as the more strategic aspects of the role
+* Stellar research skills
+
+
+## Nice-to-haves
+* Successfully led a small team (2-5 team members) and delivered on promises
+* Previous experience of initiating a new initiative and growing it
+* Strong skills in project management and project ownership
```
Comment (trentmc) [inline df.md]: I don't understand the three lines above. The role is for a software developer. Yet here it's around running / leading a project. Not appropriate here, and a very different skillset than sw dev. 

Recommend deleting the lines.

File: df.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+* Previous experience working in agile environments
+* Previous experience in working on Open source projects
```
Comment (trentmc) [inline df.md]: This list is missing out on the nice list of "nice to haves" from jobs/sr-dev.md. I recommend bringing those back, they are relevant and will get applicants excited about the position.

I've copied them here, and tuned a bit for DF. The latter point is especially important to keep, without it people might inadvertently forget these are "nice to haves" (it's a psychology thing)

- 2+ years Python 
- 1+ years CI/CD experience (e.g. Travis, tox)
- You’re obsessed with blockchain. You yield farm in your spare time and engage in crypto Twitter
- You’re an excellent writer.
- Preferred work hours: 9-5ish on European time zones
- We don’t expect you to be an expert in all of these. Rather, we’d love to see that you’re interested in them, perhaps know one or two as a starting point, and most importantly, that you’ll learn more about them as you grow with us professionally and personally.

**Review by trentmc — CHANGES_REQUESTED** [review]

File: README.md [hunk 10]
```diff
@@ -2,6 +2,7 @@
 
 - 🐬 [Senior Blockchain Developer](/sr-dev.md)
 - 🐙 [Application Engineer](/ae.md)
+- 🦈 [Senior Product Developer - Data Farming](/df.md)
```
Comment (trentmc) [inline README.md]: "Product" isn't needed. And "Blockchain" helps attract actual blockchain developers, especially due to keyword searches.

Therefore I recommend changing the line to:
"Senior Blockchain Developer - Data Farming"

**Review by trentmc — APPROVED** [review]

**Pull request merged**